package.json

```diff
@@ -43,23 +43,22 @@
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.24.1",
+    "eslint": "^4.4.0",
     "eslint-config-airbnb": "^15.1.0",
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.1.0",
-    "eslint": "^4.4.0",
     "pre-commit": "^1.1.2",
-    "react": "^15.6.1",
```

Comment:
On a clean checkout `npm install` don't install the peer dependencies so when you run `npm run lint` it's complaining that `react` is not installed.

The most sensible option, and the one commonly used by popular and well maintained projects, is to add the peer dependencies as dev dependencies as well (see for example [https://github.com/yannickcr/eslint-plugin-react/blob/master/package.json]()) so that they will be installed during development by `npm install`